src/tables.ts

```diff
@@ -434,7 +434,7 @@ export const Tables = {
       'B##': { index: 6, int_val: 13, accidental: '##' },
       BB: { index: 6, int_val: 10, accidental: 'b' },
       BBB: { index: 6, int_val: 9, accidental: 'bb' },
-      R: { index: 6, int_val: 9, rest: true }, // Rest
+      R: { index: 6, rest: true }, // Rest
```

Comment:
Are you sure this doesn't need an `Int_val`?

Reply:
A rest I believe has no pitch. This is also the way to identify that it is not a note like the X.

Reply:
`Music.getNoteValue` must throw an error if it is closed with `R`. There is even a test for that. This made me realised that rests have no value.